### PR TITLE
Add 1.0.3 release documentation.

### PR DESCRIPTION
**Pull request by leecow**
1.0.3.md - realease notes
1.0.3-manifest.md
Add 1.0.3 release notes link to root readme.

### CODE REVIEW

File: release-notes/1.0/1.0.3-manifest.md [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+Updated packages at the top of the table and marked with an *.
+
+| NuGet Package                                                       | Package Version   |
+| ------------------------------------------------------------------- | :---------------: |
+| Microsoft.NETCore.App                                               | 1.0.3*            |
+| NETStandard.Library                                                 | 1.6.0             |
+
+## ASP.NET Core
+
+All packages updated.
```
Comment (svick) [inline release-notes/1.0/1.0.3-manifest.md]: All packages updated, but none changed their version? And the same for EF.

File: release-notes/README.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # Release Notes
 
+[1.0.3 released 12/13/1016](1.0/1.0.3.md)
```
Comment (nil4) [inline release-notes/README.md]: Did you mean 12/13/**2016**?

File: release-notes/1.0/1.0.3-manifest.md [hunk 3]
```diff
@@ -0,0 +1,201 @@
+# .NET Core December 2016 Update for .NET Core 1.0 mainfest
```
Comment (Thaoden) [inline release-notes/1.0/1.0.3-manifest.md]: Typo: Should be "manifest"

File: release-notes/README.md (hunk 2 above)
Comment (richlander) [inline release-notes/README.md]: Oh no. We built a time machine!

Comment (nil4) [inline release-notes/README.md]: Could the time machine be turned on once more to retroactively fix the merge to master? /cc @leecow @blackdwarf 

> https://github.com/dotnet/core/blob/master/release-notes/README.md#release-notes
> 1.0.3 released 12/13/**1016**

**Pull request merged**